packages/aws-solution-docs/README.md

```diff
@@ -0,0 +1 @@
+TODO
```

Comment:
Please include a README.md for this package as this is what will be rendered on the documentation site.